marlowe/cli-doc/simple-test.sh

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env bash
+
+
+# Simple end-to-end test of `marlowe-cli`.
+#
+#
+# The environment variable CARDANO_NODE_SOCKET_PATH must be set.
+#
+# The following tools must be on the path:
+#   marlowe-cli
+#   cardano-cli
+#   sed
+#   jq
+#   timeout
+#
+# This script exits with an error value if the end-to-end test fails.
+
+
+set -ex
+
+
+# Select the network.
+
+NETWORK=testnet
+MAGIC_FLAG=--testnet-magic
+MAGIC_NUM=1097911063
+
+if [ -z "$CARDANO_NODE_SOCKET_PATH" ]
+then
+  export CARDANO_NODE_SOCKET_PATH=$PWD/$NETWORK.socket
+fi
```

Comment:
There should be instructions about setting `CARDANO_NODE_SOCKET_PATH` and perhaps also about setting the network magic.

Reply:
@bwbush [cardano docs](https://developers.cardano.org/docs/get-started/running-cardano/#querying-the-cardano-blockchain) suggests to set it on the bash profile

Reply:
Fixed in b780036e8acf30207c00cce6606c6299fff8bb2e.